sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplateTest.java

```diff
@@ -18,6 +24,99 @@ void testGetObjRef(){
         String objRef = resumedDataTemplate.getObjRef("IED","LDTM");
         assertEquals(expected,objRef);
     }
+
+    @Test
```

Comment:
This comment is for all following tests : naming should show test intention and it is not the case here, try to focus on the purpose more than the method name. A good convention is something like : 'should_do_something_when_action_happening', and snake case make long method names  more readable.

Reply:
I totally agree wit your comment. I tried to follow the conventions already in place in the existing test files. I think it would be better to create a specific issue to rename all tests of compas-sct.